backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/vdscommands/ScratchDiskInfo.java

```diff
@@ -0,0 +1,29 @@
+package org.ovirt.engine.core.common.vdscommands;
+
+
+import org.ovirt.engine.core.common.businessentities.storage.DiskImage;
+
+public class ScratchDiskInfo {
+
+    // Created scratch disk image.
+    private DiskImage disk;
+    // Path to the scratch disk image after it was prepared.
+    private String path;
+
+    @SuppressWarnings("unused")
+    private ScratchDiskInfo() {
+    }
+
+    public ScratchDiskInfo(DiskImage scratchDisk, String scratchDiskPath) {
+        this.disk = scratchDisk;
+        this.path = scratchDiskPath;
+    }
+
+    public DiskImage getDisk() {
+        return disk;
+    }
+
+    public String getPath() {
+        return path;
+    }
```

Comment:
I think we don't need to make it serializable now that it's part of the parameters for a vds command, yet - maybe it is worth adding setters just in case?

Reply:
Done.
Added 2 missing setters.